entity-types/infra-kubernetes_pod/definition.yml

```diff
@@ -62,3 +62,33 @@ synthesis:
       created_by_kind:
         # from kube-state-metrics kube_pod_info
         entityTagNames: [k8s.createdKind]
+
+  - compositeIdentifier:
+      separator: ":"
+      attributes:
+        - k8s.cluster.name
+        - k8s.namespace.name
+        - k8s.pod.name
+    encodeIdentifierInGUID: true
+    name: pod
+    conditions:
+      # identifier attributes
+      - attribute: k8s.pod.name
+        present: true
+      - attribute: k8s.namespace.name
+        present: true
+      - attribute: k8s.cluster.name
+        present: true
+      # open telemetry
+      - attribute: newrelic.source
+        value: 'api.traces.otlp'
+      # ebpf data
+      - attribute: instrumentation.provider
+        value: "nr_ebpf_agent"
+    tags:
```

Comment:
Should we add the other metadata columns here, e.g. all the columns that are required as conditions?

Reply:
which columns are you thinking of? I don't think we want to set `protocol.name` or `trace_role`

Reply:
Ah I assume you mean these?
```
- k8s.pod.start_time
- k8s.pod.uid
- k8s.deployment.name
- k8s.node.name
```

Reply:
Added `k8s.pod.name` to the tags as per our discussion.